src/assets/src/services/api.ts

```diff
@@ -190,7 +190,7 @@ export const updateMyUser = async (user_id: number, phone_number: string) => {
 export const searchQueue = async (term: string) => {
     const resp = await fetch(`/api/queues_search/?search=${term}`, { method: "GET" });
     await handleErrors(resp);
-    return await resp.json() as QueueAttendee[];
+    return await resp.json() as ReadonlyArray<QueueBase>;
```

Comment:
I like the readonly usage - handy for keeping anyone from falling into React's traps.